app/pastes/create-paste/createPasteInteractor.js

```diff
@@ -17,6 +18,11 @@ class CreatePasteInteractor {
             return;
         }
 
+        if (request.user !== request.authorId) {
+            this.presenter.presentFailure( new ForbiddenError('You can\'t create this paste.'));
+            return;
+        }
```

Comment:
0_0
Since we have userId inside token, we do not need this id in request anymore

Reply:
btw, again I can see inappropriate naming - `request.userId` rather then `.user`